k8s/tools/deploy-pubsub-gke.sh

```diff
@@ -0,0 +1,262 @@
+sed -i -e "s/LOG_DIR = .*/LOG_DIR = $FILESTORE_MOUNT/g" $TURBINIA_CONFIG
+sed -i -e "s/MOUNT_DIR_PREFIX = .*/MOUNT_DIR_PREFIX = '\/mnt\/turbinia'/g" $TURBINIA_CONFIG
+
+#Create Google Cloud Storage Bucket
+echo "Enabling GCS cloud storage"
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable storage-component.googleapis.com
+echo "Creating GCS bucket gs://$INSTANCE_ID"
+gsutil mb -l $REGION gs://$INSTANCE_ID
+echo "Updating $TURBINIA_CONFIG config with GCS bucket configuration"
+sed -i -e "s/GCS_OUTPUT_PATH = .*/GCS_OUTPUT_PATH = 'gs:\/\/$INSTANCE_ID\/output'/g" $TURBINIA_CONFIG
+sed -i -e "s/BUCKET_NAME = .*/BUCKET_NAME = '$INSTANCE_ID'/g" $TURBINIA_CONFIG
+
+# Create main PubSub Topic/Subscription
+echo "Enabling the GCP PubSub  API"
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable pubsub.googleapis.com
+echo "Creating PubSub topic $INSTANCE_ID"
+gcloud pubsub topics create $INSTANCE_ID
+echo "Creating PubSub subscription $INSTANCE_ID"
+gcloud pubsub subscriptions create $INSTANCE_ID --topic=$INSTANCE_ID --ack-deadline=600 
+
+# Create internal PubSub PSQ Topic/Subscription
+echo "Creating PubSub PSQ Topic $INSTANCE_ID-psq"
+gcloud pubsub topics create "$INSTANCE_ID-psq"
+echo "Creating PubSub PSQ subscription $INSTANCE_ID-psq"
+gcloud pubsub subscriptions create "$INSTANCE_ID-psq" --topic="$INSTANCE_ID-psq" --ack-deadline=600
+
+# Update Turbinia config with PubSub parameters
+echo "Updating $TURBINIA_CONFIG with PubSub config"
+sed -i -e "s/TASK_MANAGER = .*/TASK_MANAGER = 'PSQ'/g" $TURBINIA_CONFIG
+sed -i -e "s/PUBSUB_TOPIC = .*/PUBSUB_TOPIC = '$INSTANCE_ID'/g" $TURBINIA_CONFIG
+sed -i -e "s/PSQ_TOPIC = .*/PSQ_TOPIC = '$INSTANCE_ID-psq'/g" $TURBINIA_CONFIG
+
+# Enable Stackdriver Logging and Stackdriver Traceback
+echo "Enabling Cloud Error Reporting and Logging APIs"
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable clouderrorreporting.googleapis.com
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable logging.googleapis.com
+echo "Updating $TURBINIA_CONFIG to enable Stackdriver Traceback and Logging"
+sed -i -e "s/STACKDRIVER_LOGGING = .*/STACKDRIVER_LOGGING = True/g" $TURBINIA_CONFIG
+sed -i -e "s/STACKDRIVER_TRACEBACK = .*/STACKDRIVER_TRACEBACK = True/g" $TURBINIA_CONFIG
+
+# Enable Prometheus
+echo "Updating $TURBINIA_CONFIG to enable Prometheus application metrics"
+sed -i -e "s/PROMETHEUS_ENABLED = .*/PROMETHEUS_ENABLED = True/g" $TURBINIA_CONFIG
+
+# Disable some jobs
+echo "Updating $TURBINIA_CONFIG with disabled jobs"
+sed -i -e "s/DISABLED_JOBS = .*/DISABLED_JOBS = $DISABLED_JOBS/g" $TURBINIA_CONFIG
+
+# Set appropriate docker image in deployment file if user specified
+if [[ ! -z "$TURBINIA_SERVER_IMAGE" || ! -z "$TURBINIA_WORKER_IMAGE" ]] ; then
+  echo "Updating deployment files with docker image $TURBINIA_SERVER_IMAGE and $TURBINIA_WORKER_IMAGE"
+  sed -i -e "s/turbinia-server/$TURBINIA_SERVER_IMAGE/g" turbinia-server.yaml
```

Comment:
Based on a quick look at these files, it looks like this will actually change a few other instances of `turbinia-server`.  Should we anchor this with some other text to make sure we're only changing the one variable?

Reply:
Added full docker image path instead to be specific